### DISCUSSION TITLE
neovide doesn't go transparent in fullscreen in wsl

### DISCUSSION THREAD

**Discussion by abd0-omar** [neovide/neovide#1916] [Q&A] [answered]
when I open neovide and it's fullscreen it gets transparent but when I press anything the background becomes black
and the transparency is working fine when not in fullscreen mode
I don't know if the problem is a windows thing or neovide

**Comment by fredizzimo** [comment, marked as answer]
Sorry for the late answer but are you running Neovide from within WSL or through native Windows with `neovide --wsl`? We only really properly support the latter. 

I tested with https://github.com/neovide/neovide/pull/1977, which includes some fixes to the transparency rendering, and with `neovide --wsl` it works properly, also in full screen.
